docs/src/main/sphinx/connector/hive-security.rst

```diff
@@ -2,107 +2,47 @@
 Hive connector security configuration
```

Comment:
For the commit message, please follow the guideline

https://cbea.ms/git-commit/

> [Use the imperative mood in the subject line](https://cbea.ms/git-commit/#imperative)